src/testing.sh

```diff
@@ -1457,20 +1483,55 @@ rlIsOSLike() {
   return 0
 }
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# rlIsOSVersion
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+: <<=cut
+=pod
+
+=head3 rlIsOSVersion
+
+    rlIsOsVersion [Num|opNum]
+
+=over
+
+=item Num|opNum
+
+Argument is based on VERSION_ID in /etc/os-release.
+Range matching can be used in the form used by C<rlIsRHEL>.
+
```

Comment:
describe also the _operator_ and the difference between '8' and '8.4', see description of rlIsRHEL; just a reference could be enough